### PR TITLE
Support custom entrypoints (PAPIv2 alpha, life sciences beta) [BA-6406]

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: centaur/src/main/resources/standardTestCases/custom_entrypoint/custom_entrypoint.wdl [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    meta {
+        # Don't allow call-caching
+        volatile: true
+    }
+    command <<<
+        echo hello world
+    >>>
+    runtime {
+        # This image has a pre-defined entrypoint so we _have_ to override it:
+        docker: "denvazh/gatling:3.2.1"
```
Comment (mcovarr) [inline centaur/src/main/resources/standardTestCases/custom_entrypoint/custom_entrypoint.wdl:20]: Is this image and tag stable enough to use as the foundation for our CI?

Comment (cjllanwarne) [reply centaur/src/main/resources/standardTestCases/custom_entrypoint/custom_entrypoint.wdl:20]: I assume so - they'd have to go and retrospectively change history on github to break this... Happy to switch if there's a more appropriate candidate?

Comment (cjllanwarne) [reply centaur/src/main/resources/standardTestCases/custom_entrypoint/custom_entrypoint.wdl:20]: This is *the* gatling perf-test tool image btw, not _just_ some random person's docker image

Comment (cjllanwarne) [reply centaur/src/main/resources/standardTestCases/custom_entrypoint/custom_entrypoint.wdl:20]: Well... maybe not an official image but it does at least seem reasonably widely-used: 
image

Comment (mcovarr) [reply centaur/src/main/resources/standardTestCases/custom_entrypoint/custom_entrypoint.wdl:20]: we've historically put images [in a repo we control](https://github.com/broadinstitute/cromwell/blob/develop/centaur/src/main/resources/standardTestCases/non_root_default_user/non_root_default_user.wdl#L8) though I'm not sure it's worth the bother... 🙂

**Pull request merged**